src/GeositeFramework/js/SidebarToggle.js

```diff
@@ -56,7 +56,7 @@
             'click': 'togglePluginVisibility'
         },
 
-        adjustToggleIcon() {
+        adjustToggleIcon: function() {
```

Comment:
Nice find. I noticed I did this in a few other places, I think just out of habit.

Reply:
Wonder if it's possible to add a linter to catch these in PRs automatically.